### ISSUE TITLE
Disable scroll on output

### ISSUE THREAD

**Issue by pveentjer** [tomszilagyi/zutty#42]
Is it possible to disable the automatic scroll on output?

I frequently run benchmarks and each benchmark generates a line per measured time unit. And while the test is running, I want to look at what happened earlier. The problem is when there is some new output, focus is lost and I move back to the newest content.

So when I'm 'attached' to the last line, there should an automatic scroll. 

But when I'm detached, I want do not want to follow the latest output.

**Comment by tomszilagyi** [comment]
No, this is currently not supported.
xterm also has this feature, so I am familiar with it.

**Comment by pveentjer** [comment]
You are interested in having this feature?

Perhaps I could add it. I have little C++ experience, but 20y of Java experience.

**Comment by pveentjer** [comment]
I think a 'top attach' makes most sense. So you just keep following the oldest messages.

**Comment by tomszilagyi** [comment]
Yes, that is clearly the easiest to implement. How does that relate to the behaviour of other terminals?

**Comment by tomszilagyi** [comment]
Although from the perspective of the user, I am afraid it could be confusing. Seeing that the output has changed, I would assume that I am looking at the latest (live) output, until I realise (bad surprise!) that I am looking at less than current output (delayed like a public stock quote). I am not a fan, unless it turns out that prominent existing terminals do this already. Even then, I think we can do better.

**Comment by pveentjer** [comment]
qterminal does a 'top attach'. Perhaps make it a config option. So 'always follow' and 'detach' and make the first the default setting.

**Comment by tomszilagyi** [comment]
Hmm, xterm seems to do the same. Strange, I would have expected it would jump to the bottom. Seems like it's okay to have this implemented with 'top attach'. I am against a further config option apart from enable/disable scroll to bottom on input (enable being the default, as is today). Let's not complicate it more than necessary.

**Comment by tomszilagyi** [comment]
One last note, in xterm this is something that (apart from the config/resource setting read on startup) can be toggled at run-time via the Ctrl-Middle click menu ("Scroll to bottom on TTY output"). That makes it especially useful (if you know about it) since I guess this is something that you would want to turn on temporarily. Obviously, Zutty is too bare-bones for that (no menu, no toolbar).

**Comment by qw3r3wq** [comment]
This sounds like scroll lock functionality, and might be a part of VT specs? Do not have keyboard with scroll lock button right now. Just curious.
